.github/workflows/update-version.yml

```diff
@@ -30,7 +30,14 @@ jobs:
       run: |
         FILE=./src/Packages/Passport/package.json
         VERSION=${{ github.event.inputs.version }}
-        jq --arg version "$VERSION" '.version = $version' "./src/Packages/Passport/package.json" > tmp.$$.json && mv tmp.$$.json "./src/Packages/Passport/package.json"
+        jq --arg version "$VERSION" '.version = $version' $FILE > tmp.$$.json && mv tmp.$$.json $FILE
+
+    - name: Replace engine sdk version string
+      id: replace_engine_sdk_version
+      run: |
+        FILE=./src/Packages/Passport/Runtime/Scripts/Private/Helpers/SdkVersionInfoHelpers.cs
+        VERSION=${{ github.event.inputs.version }}
+        sed -i "s/__ENGINE_SDK_VERSION__/$VERSION/g" $FILE
```

Comment:
This action creates a PR with the version update (in package.json and SdkVersionInfoHelpers.cs), which I then merge into the main branch to make a release. So "__ENGINE_SDK_VERSION__" can't used to find and replace.

Reply:
I'll change the replace to look for a hard coded version number instead